azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -172,6 +174,8 @@ public AzkabanWebServer(final Server server, final Props props) throws Exception
     this.executorManager = SERVICE_PROVIDER.getInstance(ExecutorManager.class);
     this.projectManager = SERVICE_PROVIDER.getInstance(ProjectManager.class);
     this.triggerManager = SERVICE_PROVIDER.getInstance(TriggerManager.class);
+    this.metricsManager = SERVICE_PROVIDER.getInstance(MetricsManager.class);
+    this.registry = new MetricRegistry();
```

Comment:
This is a consequence of the change that @kunkun-tang suggested in AzkabanCommonModule.